drift/lib/src/runtime/manager/filter.dart

```diff
@@ -9,35 +9,44 @@ class ColumnFilters<T extends Object> {
   /// Use an extention to add more filters to any column type
   ///
   /// ```dart
-  /// extension on FilterComposer<DateTime>{
-  ///  FitlerBuilder after2000() => isAfter(DateTime(2000));
+  /// extension on ColumnFilters<DateTime>{
+  ///  ComposableFilter after2000() => isAfter(DateTime(2000));
   ///}
   /// ```
-  const ColumnFilters(this.column);
+  const ColumnFilters(this.column, [this.inverted = false]);
 
   /// Column that this [ColumnFilters] wraps
   final Expression<T> column;
 
-  /// Create a filter that checks if the column is null.
-  ComposableFilter isNull() => ComposableFilter(column.isNull());
+  /// If true, all filters will be inverted
+  final bool inverted;
 
-  /// Create a filter that checks if the column is not null.
-  ComposableFilter isNotNull() => ComposableFilter(column.isNotNull());
+  /// Returns a copy of these column filters where all the filters are inverted
+  /// ```dart
+  /// myColumn.not.equals(5); // All columns that aren't null and have a value that is not equal to 5
+  /// ```
+  /// Keep in mind that while using inverted filters, null is never returned.
```

Comment:
Isn't this also true for non-inverted filters (would `myColumn.equals(5)` include null values)?

Reply:
> would myColumn.equals(5) include null values

No, that is logical: `null !=5`. But `[1,2,3,4,5,null].where((f) => f!=5)` in dart would return `[1,2,3,4,null]`, so having it documented in the user friendly manager would be helpful.


Logically:
> "Anything that isn't a 5"  

includes ANYTHING that isn't a 5 (Which includes `null` values, as well as your neighbors kitchen sink, [if you manage to get it into a SQL database])

But:
> "Give me 5"
Doesn't mean 5 OR null

When you stop and think, people realize what happening. I just think that adding it to a doc comment would be helpful for people who aren't slammed geniuses like us. 😜